meta/group-info/tr.yaml

```
@@ -48,6 +48,7 @@ languages:
   it: İtalyanca
   ja: Japonca
   ko: Korece
+  lt: litvanyaca
```

```suggestion
  lt: Litvanca
```
make sure to re-run the script after you applied this change locally